apps/infrastructure/providers/provider.py

```diff
@@ -11,22 +11,21 @@
 import terrascript.resource as resource
 from terrascript import Module
 
-from ..tf import Terraform
-from ..utils import Config
+from apps.infrastructure.tf import ROOT_DIR, Terraform
+from apps.infrastructure.utils import Config
```

Comment:
Please change the imports to be relative since we are going to move this inside domain module
```suggestion
from ..tf import ROOT_DIR, Terraform
from ..utils import Config
```